config/config_test.go

```diff
@@ -0,0 +1,53 @@
+package config_test
+
+import (
+	"os"
+
+	. "github.com/tidepool-org/platform/config"
```

Comment:
yeah the dot import is the default with ginkgo and gomega but I have the same feeling. On one of the PR's I changed this in the data repo and asked your opinion on it. I am in favour of changing